ui/app/AppLayouts/Wallet/services/dapps/WalletConnectService.qml

```diff
@@ -265,7 +273,19 @@ QObject {
 
         sdk: root.wcSDK
         store: root.store
-        supportedAccountsModel: d.supportedAccountsModel
+        supportedAccountsModel: SortFilterProxyModel {
+            objectName: "SelectedAddressModelForDAppsListProvider"
+            sourceModel: d.supportedAccountsModel
+            filters: [
+                FastExpressionFilter {
```

Comment:
Why not a simple `ValueFilter` here?

Reply:
To be adjusted, `ValueFilter` works nicely in this usecase